analyzer/codechecker_analyzer/analyzers/analyzer_types.py

```diff
@@ -148,8 +149,11 @@ def check_supported_analyzers(analyzers):
 
     enabled_analyzers = set()
     failed_analyzers = set()
-
     for analyzer_name in analyzers:
+        analyzer_name, analyzer_version = analyzer_name.split('=', 1) \
```

Comment:
We should consider using the same format as other tools are supporting when providing version info. For example, in requirements.txt you can write this: `my_dependency==1.2.3`. In the future `>` could be used too, but for now equality check is enough.